paymentsheet/src/main/res/values-b+es+419/strings.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<resources>
+    <string name="stripe_paymentsheet_ideal_bank">Banco iDEAL</string>
+    <string name="stripe_paymentsheet_payment_method_bancontact">Bancontact</string>
```

Comment:
This already exists, I think we can use :
```
<string name="stripe_paymentsheet_payment_method_bancontact" tools:ignore="MissingTranslation">@string/Bancontact</string>
<!-- Title for Sepa Debit payment method -->
<string name="stripe_paymentsheet_payment_method_sepa_debit" tools:ignore="MissingTranslation">@string/SEPA_Debit</string>
<!-- Title for Sofort payment method -->
<string name="stripe_paymentsheet_payment_method_sofort" tools:ignore="MissingTranslation">@string/Sofort</string>
</resources>

```

Reply:
You mean not having the translated values?
I agree for iDEAL, Sofort and Bancontact, they were sent for translation but they are brand names that should remain the same.

Reply:
If they are not to be translated then I think they need to go in: donottranslate.xml   See: http://tools.android.com/recent/non-translatablestrings